src/components/Model3DViewer/Model3DViewer.tsx

```diff
@@ -0,0 +1,202 @@
+import 'regenerator-runtime';
```

Comment:
If you import this in the code it has to be a non dev-dependency

Reply:
moved to jest configuration cause this uses only for tests purposes.